app/views/pages/home/_signin.html.haml

```diff
@@ -1,11 +1,15 @@
 %h2.govuk-heading-m= t('.title')
 
-%p= t('.description')
-
-- if ENV['FEATURE_SIGN_IN_ALERT'].to_s == 'true'
+- if ENV['FEATURE_SIGN_IN_ALERT'].to_s == 'true' && !authenticated?
```

Comment:
We have a generic way to manage these feature flags, look at the code for `EmailAlertsFeature`

Reply:
feature flag removed